spec/factories/users.rb

```diff
@@ -3,5 +3,7 @@
     email
     password { '12345678' }
     gender { 'female' }
+
+    after(:create, &:confirm)
```

Comment:
This is a valid option.

Another option is add an attribute like 
```
confirmed_at      { 3.hours.ago }
```
I think the second adds less complexity than an `after_create`

Reply:
Agree with setting the attributes directly, less DB calls